### DISCUSSION TITLE
Allow SHIFT Key to elevate placing track designs

### DISCUSSION THREAD

**Discussion by Superjustinbros** [OpenRCT2/OpenRCT2#15642] [Feature requests] [answered]
Currently you can do this with placing station platforms, scenery, shops/stalls, but not track designs.

**Comment by Gymnasiast** [comment]
A very much related thing was raised in #2856, allowing track designs to be placed underground. Since that fits neatly in this feature, the feature requests have been merged.

Here is the original post by @hiandbye7:

So, you probably now this problem. You can save track designs like these two:
<img src="https://cloud.githubusercontent.com/assets/15168206/12718537/76b26806-c8ef-11e5-9ceb-12a3097c31bd.png" width="400" />
<img src="https://cloud.githubusercontent.com/assets/15168206/12718539/76db20ca-c8ef-11e5-96d3-c18f5acfa0dc.png" width="400" />
But there is no way to place them underground again:
<img src="https://cloud.githubusercontent.com/assets/15168206/12718538/76d2b64c-c8ef-11e5-8696-ed9f1f02ff67.png" width="400"/>
<img src="https://cloud.githubusercontent.com/assets/15168206/12718540/76dd5e9e-c8ef-11e5-9249-90d38eefb892.png" width="400"/>

**Comment by WhenPenguinsAttack** [comment]
Sorry to bump such an old feature request, but has any progress been made on this?

**Comment by IntelOrca** [comment]
@WhenPenguinsAttack no, perhaps you could recruit some more developers.

**Comment by fromthere** [comment]
since RCT does not allow partial underground tracks, putting design underground might be tricky. saving ground level information(setting base ground level?) might useful.

**Comment by hytacker** [comment]
It's also not possible to raise a footpath with shift. If you want to start in the air you have to begin on the ground and build towards the desired height.

**Comment by 733737** [comment]
the control and shift keys by default allow sprite stacking or matching and height adjustment
![dfxcvdxc](https://user-images.githubusercontent.com/94884086/153312242-c0fb0d11-baf2-455f-9eeb-16aab77cd781.png)
when you place a ride, it matches the surface level, I think you should have the option to match sprites or adjust height when placing a ride... it has been proven that you can ignore the surface height when clearance checks are disabled, so placing a prebuilt underground theoretically should be trivial

**Comment by AaronVanGeffen** [comment, marked as answer]
This is being implemented in #22669.
